sources/packages/web/src/components/students/modals/ConfirmEditApplication.vue

```diff
@@ -6,11 +6,37 @@
       sure you want to proceed?
     </template>
     <template #content v-else>
-      Any edits made to your application may result in a new assessment,
-      potentially delaying your application. Are you sure you want to proceed?
+      <v-checkbox
```

Comment:
As per the technical, we must create an exclusive model for the particular scenario which happens when submit button is clicked post editing an application. 

![image](https://github.com/user-attachments/assets/9dd186bb-b31c-4006-a9d2-51d3952bfd6d)

And also as per technical AC, a new modal can be created by re-using the `ConfirmModal.vue`

Here is a sample implementation of confirm modal [Confirm Modal Sample Impl](https://github.com/bcgov/SIMS/blob/099ad22c9dabc46accd4f1525ba8324847eb7e6a/sources/packages/web/src/components/common/ConfirmEnrolment.vue#L17)